packages/cli/generation/ir-generator/src/examples/ExampleGenerator.ts

```diff
@@ -795,21 +913,28 @@ export class ExampleGenerator {
         }
     }
 
-    private generateExampleTypeReferenceOptional(typeReference: TypeReference, depth: number): ExampleTypeReference {
+    private generateExampleTypeReferenceOptional(
+        typeReference: TypeReference,
+        depth: number
+    ): ExampleTypeReference | undefined {
         const exampleTypeReference = this.generateExampleTypeReference(typeReference, depth);
+        if (exampleTypeReference == null) {
+            return undefined;
+        }
         return {
             jsonExample: exampleTypeReference.jsonExample,
-            shape: ExampleTypeReferenceShape.container(
-                ExampleContainer.optional({
-                    optional: exampleTypeReference,
-                    valueType: typeReference
-                })
-            )
+            shape: exampleTypeReference.shape
```

Comment:
Do you not need to wrap this like before?

Reply:
this file is going to completely deleted now